src/components/user/SettingSection.tsx

```diff
@@ -0,0 +1,17 @@
+import { Box, BoxProps, Typography } from '@mui/material';
+import ParDivider from 'components/styled/ParDivider';
+import { FC } from 'react';
+
+export const SettingSection: FC<
+  BoxProps & { title: string; containerProps?: BoxProps }
+> = ({ title, containerProps, children }) => {
+  return (
+    <Box sx={{ display: 'flex', flexDirection: 'column' }}>
+      <Typography variant="h3" color="primary.dark">
+        {title}
+      </Typography>
+      <ParDivider sx={{ width: '100%', marginBottom: '1rem' }} />
+      <Box {...containerProps}>{children}</Box>
```

Comment:
sx={{ overflow: 'hidden' }}, I think it's much better this way, or write for every single in the props

Reply:
![image](https://user-images.githubusercontent.com/60768837/185289046-94ed15e3-f4d4-403b-9d09-d90e7bf12f50.png)
screen with scroll bar